common/src/main/java/com/wynntils/features/utilities/TranslationFeature.java

```diff
@@ -50,13 +50,29 @@ public class TranslationFeature extends Feature {
     public final Config<TranslationService.TranslationServices> translationService =
             new Config<>(TranslationService.TranslationServices.GOOGLEAPI);
 
+    @Persisted
+    public final Config<String> modelName = new Config<>("gpt-4o-mini");
```

Comment:
Since OpenAI offers various models, having the model name as a string can lead to potential typos by the user. I'd suggest using an enum for the model names.

Reply:
And then we come here and update it bi-weekly when new models release? I understand you want autocompletion of some sort, but this should just stay as is.

Reply:
> And then we come here and update it bi-weekly when new models release? I understand you want autocompletion of some sort, but this should just stay as is.

Some third-party manufacturers provide many model calls based on the OPENAI API standard, including Gemini, Claude, etc. I think it is too complicated to enumerate them.

https://www.flowus.cn/liaox/share/49604148-886b-4fe1-8b69-c64e42f60423

Reply:
> > And then we come here and update it bi-weekly when new models release? I understand you want autocompletion of some sort, but this should just stay as is.
> 
> Some third-party manufacturers provide many model calls based on the OPENAI API standard, including Gemini, Claude, etc. I think it is too complicated to enumerate them.
> 
> https://www.flowus.cn/liaox/share/49604148-886b-4fe1-8b69-c64e42f60423

@DevChromium